### PR TITLE
update docker/docker package version

### PR DESCRIPTION
**Pull request by snowhork**
This PR is for updating docker/docker package in go.mod.

`v0.7.3` is too old, so some secury check fails. (In our team, we use yamory (https://yamory.io/) (japanese))

### CODE REVIEW

**Comment by snowhork** [comment]
please review @gianarb @alde @mraerino

**Comment by snowhork** [comment]
any updates? @gianarb

**Comment by kernle32dll** [comment]
Actually, this was a massive downgrade!

Take a closer look at the version number as it was: `v0.7.3-0.20190506211059-b20a14b54661`. The referenced version was actually commit hash `b20a14b54661` (https://github.com/moby/moby/commit/b20a14b54661), not `0.7.3`.

The reason for this cryptic version number is the way go modules handle major versions. In short - docker should be importable via e.g. `github.com/moby/moby/13` - **but its not**

So - this PR actually downgraded to `v1.13.1` (https://github.com/moby/moby/releases/tag/v1.13.1), which is a **considerable** downgrade.

TL;DR; - this should be reverted **ASAP**!

Side note: This indirectly causes https://github.com/testcontainers/testcontainers-go/issues/238
